randovania/games/samus_returns/logic_database/Area 7.txt

```diff
@@ -1532,7 +1588,10 @@ Extra - asset_id: collision_camera_019
   * Layers: default
   * Tunnel to Robot Regime/Tunnel to Spider Boost Tunnel North
   > Tunnel to Robot Retreat (Top)
-      Power Bomb ≥ 2 and Spider Boost (Beginner) and Can Spider Boost
+      All of the following:
+          Spider Boost (Beginner) and Can Spider Boost
+          # Break Bomb block at the left
+          Power Bomb ≥ 2 or Lay Bomb
```

Comment:
While Bombs does make sense and would be "valid", the Bomb just saves one PB and isn't really necessary. Also I don't think the generator will ever pick that path since PBs are required to cross regardless.